account_statement_import_txt_xlsx/tests/fixtures/sample_statement_en_with_header.csv

```diff
@@ -1,15 +1,15 @@
-Posting date from;;;;;;
```

Comment:
This file worked before, why you needed to update it?

Reply:
I based the one with the headers (sample_statement_en_with_header.csv) on the basic one (sample_statement_en.csv) and also to fit with the sample mapping